src/styles/stars.scss

```
@@ -0,0 +1,44 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: no-repeat url(/src/images/star.svg) center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--1 {
+    & .stars__star:nth-child(1) {
+      background-image: url(/src/images/star-active.svg);
+    }
+  }
+
+  &--2 {
+    & .stars__star:nth-child(-n + 2) {
+      background-image: url(/src/images/star-active.svg);
+    }
+  }
+
+  &--3 {
+    & .stars__star:nth-child(-n + 3) {
+      background-image: url(/src/images/star-active.svg);
+    }
+  }
+
+  &--4 {
+    & .stars__star:nth-child(-n + 4) {
+      background-image: url(/src/images/star-active.svg);
+    }
+  }
+
+  &--5 {
+    & .stars__star:nth-child(-n + 5) {
```

```suggestion
  &--1 &__star:nth-child(-n + 1),
  &--2 &__star:nth-child(-n + 2),
  &--3 &__star:nth-child(-n + 3),
  &--4 &__star:nth-child(-n + 4),
  &--5 &__star:nth-child(-n + 5) {
```
I mean you can do like this, or even do it in a loop in `scss`